languages/fsharp/exercises/concept/arrays/.docs/instructions.md

```diff
@@ -0,0 +1,58 @@
+You're an avid bird watcher that keeps track of how many birds have visited your garden in the last seven days.
+
+You have six tasks, all dealing with the numbers of birds that visited your garden.
+
+### 1. Check what the counts were last week
+
+For comparison purposes, you always keep a copy of last week's counts nearby, which were: 0; 2; 5; 3; 7; 8 and 4. Define the `lastWeek` binding that contains last week's counts:
+
+```fsharp
+lastWeek
+// => [| 0; 2; 5; 3; 7; 8; 4 |]
+```
+
+### 2. Check how many birds visited yesterday
```

Comment:
Any reason for having yesterday in the F# version, but today in the C# one?

Reply:
Looks like there's a fair amount of divergence between the C# and F# versions actually

Reply:
There are indeed a couple of different between the exercises. Here are the reasons for some of the changes:

- Using functions instead of a class: because F# is a functional-first language (this is the most obvious change)
- Using yesterday instead of today: this will prevent the student from using `Array.last` and instead think about how indexes work in arrays (being zero-based)
- Make `incrementTodaysCount` return the updated array instead of `unit`: I wanted to not introduce the `unit` Concept at this point, so I tried to solve this by requiring the array to be returned